soundcrate/src/components/RegisterForm.js

```diff
@@ -27,6 +32,15 @@ export default function RegisterForm() {
             setError("Email is invalid");
             return;
         }
+        if (!isValidPassword(password)){
+            setError("Password must include:\n" +
+                "A minimum of 1 lower case letter\n" +
+                "A minimum of 1 upper case letter\n" +
+                "A minimum of 1 numeric character\n" +
+                "A minimum of 1 special character\n"
```

Comment:
Could we add another line to here specifying that it needs to include at least 8 characters total?

Reply:
fs